.pre-commit-config.yaml

```diff
@@ -8,7 +8,7 @@ repos:
     hooks:
     -   id: isort
 -   repo: https://github.com/ambv/black
-    rev: stable
+    rev: 19.10b0
```

Comment:
Please revert this unrelated change

Reply:
But the checks are failing if i don't add this file!

Reply:
You need to remove the change and then run pre-commit to format your files.

Reply:
Ok...it's done.